packages/eslint-config/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[Makefile]
```

Comment:
Maybe useless for us?

Reply:
You are right, I will delete this file in next version.

Reply:
> You are right, I will delete this file in next version.

`.editorconfig` is great, just `[*.md] [Makefile]` makes no sense for us (maybe...).

Reply:
Sorry, it's my fault, I don't delete the `.editorconfig` in `<root>` , here are another `.editorconfig` in `packages/eslint-config` folder, I will delete that one.

Reply:
SD

Reply:
...